[32mINFO    [0m pytest_operator.plugin:plugin.py:675 Connecting to existing model microk8s-localhost:test on unspecified cloud
[32mINFO    [0m integration.high_availability.conftest:conftest.py:68 Deploying mysql-k8s and scaling to 3 units
[32mINFO    [0m juju.model:model.py:2284 Deploying local:mysql-k8s-0
[32mINFO    [0m juju.model:model.py:3211 Waiting for model:
  mysql-k8s/0 [allocating] waiting: installing agent
  mysql-k8s/1 [allocating] waiting: installing agent
  mysql-k8s/2 [allocating] waiting: installing agent
[32mINFO    [0m juju.model:model.py:3211 Waiting for model:
  mysql-k8s/0 [executing] maintenance: installing charm software
  mysql-k8s/1 [executing] maintenance: installing charm software
  mysql-k8s/2 [executing] maintenance: installing charm software
[32mINFO    [0m juju.model:model.py:3211 Waiting for model:
  mysql-k8s/0 [executing] waiting: waiting to get cluster primary from peers
  mysql-k8s/1 [executing] active: Primary
  mysql-k8s/2 [idle] waiting: waiting to get cluster primary from peers
[32mINFO    [0m juju.model:model.py:3211 Waiting for model:
  mysql-k8s/0 [executing] active: 
  mysql-k8s/1 [idle] active: Primary
  mysql-k8s/2 [idle] maintenance: joining the cluster
[32mINFO    [0m integration.high_availability.conftest:conftest.py:71 Deploying mysql-test-app
[32mINFO    [0m juju.model:model.py:2284 Deploying ch:amd64/jammy/mysql-test-app-63
[32mINFO    [0m juju.model:model.py:3211 Waiting for model:
  mysql-test-app/0 [allocating] waiting: installing agent
[32mINFO    [0m juju.model:model.py:3211 Waiting for model:
  mysql-test-app/0 [idle] waiting: 
[32mINFO    [0m integration.high_availability.conftest:conftest.py:74 Relating mysql-k8s with mysql-test-app
[33mWARNING [0m juju.model:model.py:1666 relate is deprecated and will be removed. Use integrate instead.
[32mINFO    [0m juju.model:model.py:3211 Waiting for model:
  mysql-k8s/0 [idle] active: 
  mysql-k8s/1 [idle] active: Primary
  mysql-k8s/2 [idle] active: 
  mysql-test-app/0 [idle] waiting: 
[32mINFO    [0m integration.high_availability.conftest:conftest.py:29 Clearing continuous writes
[32mINFO    [0m integration.high_availability.conftest:conftest.py:31 Starting continuous writes
[32mINFO    [0m integration.high_availability.high_availability_helpers:high_availability_helpers.py:273 Deploying Chaos Mesh
[32mINFO    [0m integration.high_availability.high_availability_helpers:high_availability_helpers.py:283 Ensure chaos mesh is ready
[32mINFO    [0m integration.conftest:conftest.py:20 Getting credentials for the MySQL cluster
[32mINFO    [0m integration.high_availability.test_self_healing:test_self_healing.py:311 Ensuring that there are 3 online mysql members
[32mINFO    [0m integration.high_availability.test_self_healing:test_self_healing.py:316 Ensuring that all instances have incrementing continuous writes
[32mINFO    [0m integration.high_availability.high_availability_helpers:high_availability_helpers.py:505 max_written_value=3546 on unit mysql-k8s/0
[32mINFO    [0m integration.high_availability.high_availability_helpers:high_availability_helpers.py:505 max_written_value=3549 on unit mysql-k8s/1
[32mINFO    [0m integration.high_availability.high_availability_helpers:high_availability_helpers.py:505 max_written_value=3552 on unit mysql-k8s/2
[32mINFO    [0m integration.high_availability.test_self_healing:test_self_healing.py:324 Creating networkchaos policy to isolate instance mysql-k8s/1 from the cluster
[32mINFO    [0m integration.high_availability.test_self_healing:test_self_healing.py:331 Wait until MySQL GR actually detects isolated instance
[32mINFO    [0m integration.high_availability.test_self_healing:test_self_healing.py:348 Ensure all units have incrementing continuous writes
[32mINFO    [0m integration.high_availability.high_availability_helpers:high_availability_helpers.py:505 max_written_value=14016 on unit mysql-k8s/0
[32mINFO    [0m integration.high_availability.high_availability_helpers:high_availability_helpers.py:505 max_written_value=14018 on unit mysql-k8s/2
[32mINFO    [0m integration.high_availability.test_self_healing:test_self_healing.py:353 Remove networkchaos policy isolating instance from cluster
[32mINFO    [0m integration.high_availability.test_self_healing:test_self_healing.py:357 Wait until returning instance enters recovery
[32mINFO    [0m integration.high_availability.test_self_healing:test_self_healing.py:361 Wait until returning instance become active
[32mINFO    [0m integration.high_availability.test_self_healing:test_self_healing.py:366 Wait until all units are online
[32mINFO    [0m integration.high_availability.test_self_healing:test_self_healing.py:371 Ensure isolated instance is now secondary
[32mINFO    [0m integration.high_availability.test_self_healing:test_self_healing.py:380 Ensure there are 3 online mysql members
[32mINFO    [0m integration.high_availability.test_self_healing:test_self_healing.py:385 Ensure all units have incrementing continuous writes
[32mINFO    [0m integration.high_availability.high_availability_helpers:high_availability_helpers.py:505 max_written_value=17685 on unit mysql-k8s/0
[32mINFO    [0m integration.high_availability.high_availability_helpers:high_availability_helpers.py:505 max_written_value=17688 on unit mysql-k8s/1
[32mINFO    [0m integration.high_availability.high_availability_helpers:high_availability_helpers.py:505 max_written_value=17691 on unit mysql-k8s/2
[32mINFO    [0m integration.high_availability.conftest:conftest.py:47 Destroying chaos mesh
[32mINFO    [0m integration.high_availability.conftest:conftest.py:36 Clearing continuous writes
[32mINFO    [0m pytest_operator.plugin:plugin.py:790 Model status:

Model  Controller          Cloud/Region        Version  SLA          Timestamp
test   microk8s-localhost  microk8s/localhost  3.6.1    unsupported  01:36:22Z

App             Version                  Status  Scale  Charm           Channel      Rev  Address         Exposed  Message
mysql-k8s       8.0.39-0ubuntu0.22.04.1  active      3  mysql-k8s                      0  10.152.183.50   no       
mysql-test-app  0.0.2                    active      1  mysql-test-app  latest/edge   63  10.152.183.211  no       Last written value=16300

Unit               Workload  Agent  Address      Ports  Message
mysql-k8s/0*       active    idle   10.1.251.77         Primary
mysql-k8s/1        active    idle   10.1.251.76         
mysql-k8s/2        active    idle   10.1.251.75         
mysql-test-app/0*  active    idle   10.1.251.78         Last written value=16300

[32mINFO    [0m pytest_operator.plugin:plugin.py:796 Juju error logs:

unit-mysql-k8s-0: 01:32:48 ERROR unit.mysql-k8s/0.juju-log Failed to get cluster status for test_cluster
unit-mysql-k8s-1: 01:34:56 ERROR juju.worker.dependency "api-caller" manifold worker returned unexpected error: [33cc6f] "unit-mysql-k8s-1" cannot open api: cannot resolve "controller-service.controller-microk8s-localhost.svc.cluster.local": lookup controller-service.controller-microk8s-localhost.svc.cluster.local: i/o timeout
unit-mysql-k8s-1: 01:35:03 ERROR juju.worker.dependency "api-caller" manifold worker returned unexpected error: [33cc6f] "unit-mysql-k8s-1" cannot open api: cannot resolve "controller-service.controller-microk8s-localhost.svc.cluster.local": lookup controller-service.controller-microk8s-localhost.svc.cluster.local: i/o timeout
unit-mysql-k8s-1: 01:35:09 ERROR juju.worker.dependency "api-caller" manifold worker returned unexpected error: [33cc6f] "unit-mysql-k8s-1" cannot open api: unable to connect to API: dial tcp 10.152.183.238:17070: connect: no route to host

[32mINFO    [0m pytest_operator.plugin:plugin.py:862 Forgetting main...